policybot/pkg/testresults/testresults.go

```diff
@@ -0,0 +1,337 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Take in a pr number from path "istio-prow/pr-logs/pull/istio-istio" and examine the pr
```

Comment:
This path must be configurable, so please remove it from the comment. You can just say "from blob storage".
examine -> examines